website/docs/tutorials/managing-access.md

```diff
@@ -123,19 +123,150 @@ const accessByAgent = unstable_getAgentAccessAll(litDatasetWithAcl);
 
 ## Changing who has access
 
-:::caution
-
-This section is still being written.
-
-:::
+To be able to _change_ who has access to a specific Resource, you will need to know a little bit
+more about how Access Control works in Solid:
 
 ### Two types of ACL
 
 A Resource _can_ have an ACL that applies to just that Resource. However, if no such ACL exists, the
-Pod server will fall back to the ACL of its [Container](../glossary#container) — or its Container's
+Pod server will [fall back](../glossary#fallback-acl) to the ACL of its [Container](../glossary#container) — or its Container's
 Container's, or its Container's Container's Container's, etc.
 
 Thus, an ACL can control both access to a specific Resource or Container directly, and provide
 _default_ access rules: those that apply to the _children_ of the applicable Container when it
 serves as their fallback ACL. Note that the Container's _Resource_ access rules will _not_ apply to
 its children.
+
+### Modifying the ACL
+
+To modify access to a Resource, you will need to obtain its ACL. Assuming you have fetched the
+Resource using
+[`unstable_fetchLitDatasetWithAcl`](../api/modules/_litdataset_#unstable_fetchlitdatasetwithacl),
+you can call
+[`unstable_getResourceACL`](../api/modules/_acl_acl_#unstable_getresourceacl) to obtain its ACL — or
+`null` if it does not exist.
+
+If it exists, that's great, and you can move on to the next section. You can create a new ACL if it
+does not exist yet, but be aware that doing so will result in any access that is currently granted
+to the Resource via the default access rules of its fallback ACL, as described in the previous
+section, will no longer apply.
+
+A new empty ACL can be initialised using
+[`unstable_createAcl`](api/modules/_acl_acl_#unstable_createacl), given that the current user has
+the rights to do so. However, keep in mind that this ACL will override any ACL that currently
```

Comment:
```suggestion
the rights to do so (_i.e._ Control access on the target Resource). However, keep in mind that this ACL will override any ACL that currently
```

Reply:
Fixed in 86578da32ed01aa15ec8240cc2e219ae0fb2d9ea (sorry, I did not italicise the "i.e." :P ).